pandas/core/arrays/period.py

```diff
@@ -408,7 +412,10 @@ def __arrow_array__(self, type=None):
                     f"Not supported to convert PeriodArray to '{type}' type"
                 )
 
-        period_type = ArrowPeriodType(self.freqstr)
+        if self.freqstr == "ME":
+            period_type = ArrowPeriodType("M")
+        else:
+            period_type = ArrowPeriodType(self.freqstr)
```

Comment:
can we use the `freq_to_period_freqstr` function here too?

Reply:
Yes, I think we can use here the function `freq_to_period_freqstr`. I made the changes.